h/admin/views/features.py

```diff
@@ -29,6 +32,15 @@ def features_save(request):
                 setattr(feat, attr, True)
             else:
                 setattr(feat, attr, False)
+        for cohort in request.db.query(models.FeatureCohort).all():
```

Comment:
This function is now getting to the point where it needs splitting up into some helpers, I think. Leave it for now, but in future maybe consider factoring stuff out when it gets this busy.